README.md

```diff
@@ -229,6 +230,12 @@ architecture distributed over the application modules. The _controller_ in the a
 task-based and executes all tasks on a background thread to avoid any possibility of blocking
 the Android UI thread.
 
+#### MVVM
+
+Newer application modules, roughly follows an [MVVM](https://en.wikipedia.org/wiki/Model%E2%80%93view%E2%80%93viewmodel) architecture.
```

Comment:
`s/follows/follow/`

:stuck_out_tongue:

Reply:
😂  https://youtu.be/e1ghnsE6B0k?t=7

Reply:
:laughing: